### PR TITLE
Improved Export Methods

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Comment by akaszynski** [comment]
It's weird that we're enforcing coverage within unit tests. Let me ignore that.

**Comment by maxcapodi78** [comment]
> It's weird that we're enforcing coverage within unit tests. Let me ignore that.

correct. Thank you

File: pyaedt/modules/PostProcessor.py [hunk 1]
```diff
@@ -1928,7 +1928,7 @@ def export_image(self, full_path=None, width=1920, height=1080, orientation="iso
         ----------
 
         >>> oModule.ExportPlotImageToFile
-        >>> oModule.ExportPlotImageWithViewToFile
```
Comment (Alberto-DM) [inline pyaedt/modules/PostProcessor.py:1931]: I'd leave it because it is still used.

**Review by Alberto-DM — APPROVED** [review]
LGTM

**Pull request merged**